rpm/0001-Disable-documentation-further.patch

```diff
@@ -0,0 +1,61 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Pekka Vuorela <pekka.vuorela@jolla.com>
+Date: Tue, 10 Jan 2023 18:00:06 +0200
+Subject: [PATCH] Disable documentation further.
+
+Suppress documentation installation, as it causes problems in our build
+
+Signed-off-by: Marko Saukko <marko.saukko@jolla.com>
+---
+ Makefile.    | 0
+ Makefile.am  | 8 ++++----
+ configure.ac | 2 +-
+ 3 files changed, 5 insertions(+), 5 deletions(-)
+ create mode 100644 Makefile.
+
+diff --git a/Makefile. b/Makefile.
+new file mode 100644
+index 00000000..e69de29b
+diff --git a/Makefile.am b/Makefile.am
+index 316109b1..3d643c32 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -2,12 +2,12 @@
+ 
+ ACLOCAL_AMFLAGS = -I m4
+ 
+-SUBDIRS = include . doc example fuzz xstc
++SUBDIRS = include . fuzz xstc
+ if WITH_PYTHON
```

Comment:
You don't need to patch this.
If you pass SUBDIR="include . fuzz xstc" to make it will override the default.

Reply:
I didn't change this more than rebasing the patch. SUBDIR sounds a bit arbitrary thing used externally so I'd rather not touch this now.

Reply:
SUBDIRS is the standard variable for sub makefiles in Automake projects, I've tested this with guile already overriding it over the commant line is fine.
In any case it is just a smaller suggestion that avoids the patch.